src/test/unit/service/publicationService.test.ts

```diff
@@ -109,4 +112,26 @@ describe('Publication service', () => {
       });
     });
   });
+
+  describe('formatSJPPressList Publication Service', () => {
+    it('should return SJP Press List', async () => {
```

Comment:
these tests are redundant, you are evaluating input `dailyCauseListData` `expect(dailyCauseListData['courtLists'].length).to.equal(1);`
It should be 
```
const response = await publicationService.calculateHearingSessionTime(dailyCauseListData);
expect(response.whatever).to.equal(dailyCauseListData.whatever)
```

**This applies to all tests in this file**

Reply:
no, these tests cannot be redundant. I am using formatSJPPressList for the tests line 116 below. line 86 is just a text change which I found while creating tests for this ticket.

Reply:
Ah I see, your function doesn't return anything you are just mutating the data passed in.
Sorry! 
Although this is considered a bad practice
https://medium.com/the-andela-way/its-bad-practice-to-reassign-function-params-c37f289a5dc4
https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/

Reply:
I think it depends if you are using same variable below in some other function or manipulating the data, then it is not a good practice. In my case, once I manipulate the data, it passed to the view.

Reply:
This is not a good practice.. publicationService.formatSJPPressList(data) is mutating data inside that function indeed. 
Also it could lead to cause future issues if a change goes unnoticed in here.

Reply:
It's done.